rush.json

```diff
@@ -154,19 +154,19 @@
       "packageName": "@typespec/eslint-config-typespec",
       "projectFolder": "packages/eslint-config-typespec",
       "reviewCategory": "production",
-      "shouldPublish": true
+      "versionPolicyName": "typespec"
     },
     {
       "packageName": "@typespec/internal-build-utils",
       "projectFolder": "packages/internal-build-utils",
       "reviewCategory": "production",
-      "shouldPublish": true
+      "versionPolicyName": "typespec"
     },
     {
       "packageName": "@typespec/spec",
       "projectFolder": "packages/spec",
       "reviewCategory": "production",
-      "shouldPublish": false
+      "versionPolicyName": "typespec"
```

Comment:
This one isn't published, I don't think there's any need to publish it either.

Reply:
Is there a way to say, bump with this policy, but don't publish?

Reply:
ah yes oops, if its not published it doesn't ask for changelog and bump version anyway

Reply:
reverted